cegs_portal/search/views/v1/reg_effects.py

```diff
@@ -107,6 +109,8 @@ def get_sig_only(value):
         options["per_page"] = int(request.GET.get("per_page", 20))
         sig_only = request.GET.get("sig_only", True)
         options["sig_only"] = get_sig_only(sig_only)
+        is_tsv = request.GET.get("is_tsv", False) == "true"
```

Comment:
I'd figured I would do something like we did for sig_only because the code in regeffects.py didn't work for both the table and tsv button. When I implement the code from regeffects.py, the table data disappears. This solution allowed both tsv button and the table data output to work.

Reply:
The question of whether or not they are asking for a tsv is answered in TemplateJsonView. There doesn't need to be another query parameter for it.

Reply:
Would you like to go over how TemplateJsonView works when you get back?

Reply:
Right. 
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/03869e95-209f-40d1-863e-098e652abec9)

Reply:
Right. But also note how in https://github.com/ReddyLab/cegs-portal/pull/171/files there is no format check in get_data. `get_data` returns the queryset by default. The pagination code was moved into `get` and `get_json` methods on FeatureEffectsView. IMO that is how this should be handled.

Reply:
Just read your comment. Pushed another commit solution that works but it seems it isn't the solution preferred. I'll try the other way.